AutocompleteClient/FW/Logic/Request/Builder/QueryItemCollection.swift

```diff
@@ -66,9 +66,20 @@ struct QueryItemCollection {
         var dict: [String: Any] = [:]
         flattenedArray.forEach { item in
             if item.value != nil {
-                dict[item.name] = item.value
+                if let value = dict[item.name] {
+                    if var arrayValue = value as? [String] {
+                        arrayValue.append(item.value!)
+                    } else {
+                        dict[item.name] = [value, item.value!]
+                    }
+                } else {
+                    dict[item.name] = item.value
```

Comment:
Originally was going to add this logic but to be I opted to instead just delete the US parameter, create another story to explore this.